app/helpers/cases_helper.rb

```diff
@@ -236,6 +236,11 @@ def action_button_for(event)
               case_cover_page_path(@case),
               id: "action--preview-cover-page",
               class: "button-secondary"
+    when :information_received
+      link_to "Information received",
+              "",
```

Comment:
Not added the link yet as its to be done in [cdpt-1125-information-received-page](url)